textattack/commands/train_model/run_training.py

```diff
@@ -25,8 +25,7 @@
 
 
 def _save_args(args, save_path):
-    """
-    Dump args dictionary to a json
+    """Dump args dictionary to a json.
```

Comment:
Did you mean to change these? I guess this is a `make format` thing? But then why was it merged without formatting?